### PR TITLE
running-coreos: update cloud-config for etcd2

### PR DESCRIPTION
**Pull request by robszumski**
@crawford Snuck a `network-online.target` commit in here for ECS

### CODE REVIEW

File: running-coreos/cloud-providers/ecs/index.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ coreos:
        Description=Amazon ECS Agent
        After=docker.service
        Requires=docker.service
+       Requires=network-online.target
```
Comment (crawford) [inline running-coreos/cloud-providers/ecs/index.md]: After=

File: running-coreos/cloud-providers/azure/index.md [hunk 2]
```diff
@@ -69,15 +69,19 @@ The most common cloud-config for Azure looks like:
...
-    # deployments across multiple cloud services will need to use $public_ipv4
-    addr: $private_ipv4:4001
-    peer-addr: $private_ipv4:7001
+    # multi-region and multi-cloud deployments need to use $public_ipv4
+    advertise-client-urls: http://$private_ipv4:2379
+    initial-advertise-peer-urls: http://$private_ipv4:2380
+    # listen on both the official ports and the legacy ports
+    # legacy ports can be omitted if your application doesn't depend on them
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://$private_ipv4:2380
```
Comment (crawford) [inline running-coreos/cloud-providers/azure/index.md]: should this also have the legacy port?

Comment (robszumski) [inline running-coreos/cloud-providers/azure/index.md]: You can't operate mixed clusters so I don't see why

**Comment by crawford** [comment]
LGTM

**Comment by crawford** [comment]
Please rebase to remove that merge commit.

**Comment by crawford** [comment]
:+1:

**Pull request merged**